### PR TITLE
Follow up of #76314

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Emergency bug fix for #76314.
`xargs` does not work correctly because of incorrect quotes.

```
!!! [0413 03:12:44] Call tree:
!!! [0413 03:12:44]  1: ./hack/update-gofmt.sh:47 find_files(...)
!!! Error in ./hack/update-gofmt.sh:31
  Error in ./hack/update-gofmt.sh:31. '((i<3-1))' exited with status 141
Call stack:
  1: ./hack/update-gofmt.sh:31 find_files(...)
  2: ./hack/update-gofmt.sh:47 main(...)
Exiting with status 1
!!! Error in ./hack/update-gofmt.sh:47
  Error in ./hack/update-gofmt.sh:47. 'xargs "${GOFMT}"' exited with status 1 127
Call stack:
  1: ./hack/update-gofmt.sh:47 main(...)
Exiting with status 1
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/cc @fejta @neolit123

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm
/priority critical-urgent

**Review by fejta — APPROVED** [review]
/lgtm
/approve

File: hack/update-gofmt.sh [hunk 1]
```diff
@@ -43,5 +43,4 @@ find_files() {
     \) -name '*.go'
 }
 
-GOFMT="gofmt -s -w"
-find_files | xargs "${GOFMT}"
+find_files | xargs gofmt -s -w
```
Comment (tedyu) [inline hack/update-gofmt.sh]: There is similar quote in verify-gofmt.sh
```
diff=$(find_files | xargs "${gofmt}" -d -s 2>&1) || true
```
Should I send out a PR with similar change ?

Comment (SataQiu) [reply hack/update-gofmt.sh]: Thanks @tedyu 
But these two are different. 
In `verify-gofmt.sh`, `${gofmt}` is the path of `gofmt`, but not a command.
I don't think we need to modify `verify-gofmt.sh`.

**Pull request merged**